x/auth/ante/basic.go

```diff
@@ -122,7 +122,7 @@ func (cgts ConsumeTxSizeGasDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, sim
 			}
 
 			// use stdsignature to mock the size of a full signature
-			simSig := legacytx.StdSignature{ //nolint:staticcheck // this will be removed when proto is ready
+			simSig := legacytx.StdSignature{ // nolint: staticcheck // SA1019: legacytx.StdSignature is deprecated
```

Comment:
Here there is a space between `notlint` and `staticcheck`, is that expected?

Reply:
I should fix it, but the linter won't care because I told it not to.